_docs/manage/features/high-availability.md

```diff
@@ -18,6 +18,22 @@ The maximum number of replicas that can be created is five, or up to the number
 of remaining nodes in the cluster. For most applications, one replica is
 sufficient.
 
+## Failure modes
+
+Failure modes specify how StorageOS scheduler should react to node issues. 
+
+There are three different modes:
+
+* **soft** is the default mode. It works together with the failure tolerance label to help decide whether volume should be writable or not. Failure tolerance specifies how many failed replicas we tolerate, defaults to (Replicas - 1) if Replicas > 0, o if we have 2 replicas it will default to 1, if we have 3 replicas, tolerance will be 2.
```

Comment:
"failed replicas we tolerate" => "failed replicas to tolerate". Avoid using "we" in docs.
Stray o in "Replicas > 0, o if"

You could display the examples in a table under each mode eg. under **soft**
storageos.feature.replicas | storageos.com/failure.tolerance | Number of failures | Volume
0 | x | 1 | Unavailable

and two more tables under hard and alwayson.

Reply:
thanks, it was a copy paste from the source :)

Reply:
Was this meant for a different comment? I still see "we" and the stray "o".